CHANGELOG.md

```diff
@@ -1,8 +1,66 @@
 # Rerun changelog
 
+
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
-## [0.18.0](https://github.com/rerun-io/rerun/compare/0.16.1...HEAD) - Ingestion speed and memory footprint
+
+## [0.18.1](https://github.com/rerun-io/rerun/compare/0.17.0...HEAD) - Bug fix
+
+Bug fixes and performance improvements for Rerun 0.18.0.
+
+### 🔎 Details
+
+### ⚠️ Breaking changes
+- Remove unused `Datatype` and `DatatypeBatch` [#7256](https://github.com/rerun-io/rerun/pull/7256)
+- Remove unused `Loggable{Batch}::arrow_field` [#7257](https://github.com/rerun-io/rerun/pull/7257)
+- Remove unused `LoggableBatch::num_instances` [#7258](https://github.com/rerun-io/rerun/pull/7258)
+- Remove unused `Loggable::extended_arrow_datatype` [#7260](https://github.com/rerun-io/rerun/pull/7260)
```

Comment:
⚠️ We shouldn't ship breaking changes in a patch release! We can just `git revert` these instead